src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -864,7 +890,10 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
             .map((dropdownId) => {
               const entry = dropdownConfigs[dropdownId];
               const btnConfigs = entry.buttonConfigs;
-              const dropdownTitle = entry.dropdownConfig.title;
+              // const dropdownTitle = entry.dropdownConfig.title;
```

Comment:
don't change dropdown title, you should change each individual button's text instead (if the button is a download button)